.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: "\U0001F41B bug"
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
I would probably use `#`/`##`/`###` headers here or if you prefer add a blank line to put the headers on their own lines. Ditto for the rest

Reply:
Done in 93a14bf9005ede9bc54a85c3f02211de898fd9f2